src_c/font.c

```diff
@@ -371,15 +371,14 @@ static PyObject *
 font_render(PyObject *self, PyObject *args)
 {
     TTF_Font *font = PyFont_AsFont(self);
-    int aa;
+    int antialias;
     PyObject *text, *final;
-    PyObject *fg_rgba_obj, *bg_rgba_obj = NULL;
+    PyObject *fg_rgba_obj, *bg_rgba_obj = Py_None;
     Uint8 rgba[] = {0, 0, 0, 0};
     SDL_Surface *surf;
-    SDL_Color foreg, backg;
-    int just_return;
+    const char *astring = "";
 
-    if (!PyArg_ParseTuple(args, "OpO|O", &text, &aa, &fg_rgba_obj,
+    if (!PyArg_ParseTuple(args, "OpO|O", &text, &antialias, &fg_rgba_obj,
```

Comment:
Could take this opportunity to add keyword arguments if you fancy?

Or just leave it for another PR.

Reply:
I'll leave it for another PR. I did an experiment with METH_FASTCALL on this, and got an additional 4% speedup for rendering text. But I'm not trying to merge that because FASTCALL is 3.7 and up, and there isn't a good way to parse keywords with it.